tests/test_zoned_datetime.py

```diff
@@ -342,6 +342,85 @@ def test_from_iso(self):
             )
         )
 
+    def test_leap_seconds_parsing(self):
+        # Leap second (60) should be parsed and normalized to 59
+        assert ZonedDateTime.parse_iso(
```

Comment:
Let's move these ISO-parsing test cases to the parametrized `TestParseIso.test_valid` test case. You can add comments above them that these are leap second test cases.